server/controllers/message.js

```diff
@@ -0,0 +1,113 @@
+const Message = require("../models/Message");
+const asyncHandler = require("express-async-handler");
+const Conversation = require("../models/Conversation");
+
+exports.getAllMessages = asyncHandler(async (req, res, next) => {
+  const messages = await Message.find({
+    conversation_id: req.params.conversationId,
+  });
+
+  if (!messages) {
+    res.status(404);
+    throw new Error("No messages found for given id");
+  }
+
+  res.status(200).json({ messages: messages });
+});
+
+exports.createMessage = asyncHandler(async (req, res, next) => {
+  const { conversation_id, body } = req.body;
```

Comment:
Can you sweep through these files and update snake-case variables to camel-case?